test/visual/test_cases/date_extrema/options

```diff
@@ -0,0 +1,17 @@
+series: [
+  {
+    y: "y",
+    label: "y"
+  }
+],
+axes: {
+  x: {
+    key: "x",
+    type: "date",
+    min: new Date('2014-01-01'),
+    max: new Date('2016-01-01'),
+    ticksFormatter: function formatDate(date) {
```

Comment:
just a comment, we don't need to change it though. you could write this with the `ticksFormat` option - it uses [d3.time.format](https://github.com/mbostock/d3/wiki/Time-Formatting) under the hood; it would look like `ticksFormat: '%m/%Y'`